examples/embed/widget.py

```diff
@@ -103,7 +103,7 @@ def update_pyramid(self):
 pop = Population()
 pop.render()
 
-tag = embed.autoload_server(pop.layout)
+tag = embed.autoload_server(session_id=pop.session.id, pop.layout)
```

Comment:
I understand why this is the way it is but I wonder if there is any additional convenience layer we could add that would free the user from having to make this coordination. `session.autoload_server(layout)`, `embed.autoload_from_cursession(layout)`?